idd/Energy+.idd.in

```diff
@@ -6850,10 +6849,31 @@ MaterialProperty:MoisturePenetrationDepth:Settings,
        \required-field
        \units dimensionless
        \type real
-  N5 ; \field Moisture Equation Coefficient d
+  N5 , \field Moisture Equation Coefficient d
        \required-field
        \units dimensionless
        \type real
+  N6 , \field Surface-layer penetration depth
```

Comment:
@Myoldmopar Hyphens are used sparingly in field names, most common being Dry-Bulb.  My preference here would be to drop the hyphens, but if you keep it, both words should be Title case, "Surface-Layer Penetration Depth".
And be sure to update the I/O ref headings to match.
IP-Units for m will default to ft. If you prefer inches, then add "\ip-units in" to these fields.